src/CommandingOfficers/CommanderAbility.java

```diff
@@ -52,18 +98,15 @@ public String toString()
    */
   protected void adjustCost()
   {
-    // Increase the cost of this ability for next time to mitigate spam and
-    // accommodate the presumably-growing battlefront.
-    // Cost grows by at least one, and at most 10% of the current cost.
-    myPowerCost += Math.max(myPowerCost*0.1, 1);
+    costBasis.numCasts += 1;
   }
```

Comment:
Is there a reason to keep  `adjustCost` around at all now that it isn't actually adjusting the cost? Seems like you could just inline the contents in `activate`.

Reply:
Ave overloads it.

Reply:
Oh, I thought I commented on that. I'm fine with the approximate fix you did; it's not like the numbers were fine-tuned for balance before.

Reply:
You totally did. I wrote that days ago; it just showed up now because I was "writing a review" until today.